libqalculatenasc/meson.build

```diff
@@ -0,0 +1,10 @@
+libqalculatenasc = static_library(
+    'libqalculatenasc',
+
+    'QalculateNasc.cc',
+    'QalculateNasc.h',
+
+    dependencies: [
+        meson.get_compiler('cpp').find_library('libqalculate'),
```

Comment:
@decathorpe 

You forgot about `cln`.

Replace this:
```
        meson.get_compiler('cpp').find_library('libqalculate'),
```
with this:
```
        dependency('cln'),
        dependency('libqalculate'),
```

This should fix builds on Fedora < 28 and EL < 8, as well as on Freedesktop 18.08 and GNOME 3.30.
https://github.com/parnold-x/nasc/pull/98#issuecomment-438664227
```
/usr/bin/ld: liblibqalculatenasc.a(libqalculatenasc_QalculateNasc.cc.o): undefined reference to symbol '_ZN3cln24cl_GV_number_init_helperD1Ev'
//usr/lib/libcln.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

```
libqalculatenasc/meson.build:1:0: ERROR:  C++ library 'libqalculate' not found
```

Reply:
Interesting ... that's the first time I read about the `cln` library. But since it fixes the problem ... :man_shrugging: